src/pages/index.tsx

```diff
@@ -30,39 +31,71 @@ const Cards = () => (
       >
         {[
           {
-            header: "DevOps, monitoring and observability",
-            content:
-              "Collect CPU, memory and storage metrics from your infrastructure and get real-time visibility into your entire stack.",
+            header: "Financial market data",
+            content: (
+              <p>
+                Store years of market tick data to identify historical trends,
+                and correlate it with orders and executed trades via{" "}
+                <Link to="/docs/reference/sql/join/#asof-join">ASOF joins</Link>.
+                Monitor market activity in real-time, compute time-series charts
+                and price candles on the fly. Open source kdb+ alternative.
+              </p>
```

Comment:
unnecesary nesting with `<p>`, because `content` is already rendered within paragraph, see [here](https://github.com/questdb/questdb.io/blob/b1be9fe35ae8bfd2f9e9574ad3a03ea1576262b8/src/pages/index.tsx#L103).

Double nesting results in such HTML:

![image](https://github.com/questdb/questdb.io/assets/4284659/b4b7afff-952f-44cd-af6b-3fbc6c07ef3c)

it also adds extra new space between paragraphs.

we can avoid double nesting by using `<React.Fragment>` instead, or it's shorthand `<>`:

```suggestion
              <>
                Store years of market tick data to identify historical trends,
                and correlate it with orders and executed trades via{" "}
                <Link to="/docs/reference/sql/join/#asof-join">ASOF joins</Link>.
                Monitor market activity in real-time, compute time-series charts
                and price candles on the fly. Open source kdb+ alternative.
              </>
```